docs/data_fields.md

```diff
@@ -0,0 +1,28 @@
+
+The main fields are the following, plus the geographical entity fields explained below:
+
+* date: the date the data point refers to
+* type: the type of data point: cases, tested, deaths, hospitalized, discharged, or recovered
+* value: the value of the data point (a count of events of a certain type)
+
+Each observation is associated with a geographical entity, identified in the following four fields.
+
+
+* city: the name of the city
+* county: county or parish or the appropriate name of the administrative subdivision below the level of state or equivalent 
+* state: state, province or region depending on jurisdiction. In general, the first administrative subdivision below the level of country.
```

Comment:
Unlike the country, we don't refer to a standard for these names. This can make joining other datasets difficult, particularly for counties in US and maybe elsewhere.

Reply:
That's true. We can align around something @hyperknot

Reply:
I see there is discussion going on as to the correct approach. In the interest of delivering this document, I will make an issue to document this after the new standards are in place

Reply:
I think this is premature while discussion is ongoing and out of scope for this PR (We can't document what is not done yet, and it's my fault for mentioning it here). I entered #601 to remind ourselves of this task. Please mention it when work on this starts.